pom.xml

```diff
@@ -124,6 +125,11 @@
 				<version>${project.version}</version>
```

Comment:
I think you can remove the dependency to 'nl.jqno.equalsverifier:equalsverifier` from mvvmfx-core because this util is only used in the tests for the validator.